android/src/main/java/net/no_mad/tts/TextToSpeechModule.java

```diff
@@ -525,4 +525,14 @@ private void sendEvent(String eventName, WritableMap params) {
                 .getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter.class)
                 .emit(eventName, params);
     }
+    
+    @ReactMethod
+    public void removeListeners(Integer count) {
+        // Keep: Required for RN built in Event Emitter Calls.
+    }
+
+    @ReactMethod
+    public void addListener(Integer count) {
```

Comment:
It should be:
`public void addListener(String eventName) {`
Otherwise application throws an error.

Reply:
@fatidian1 you raise a PR then I will merge for you